### ISSUE TITLE
New Keyboard Shortcut to Open Context Menu

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#8993]
While everything a context menu allows you to do with a cursor is available in the keyboard shortcuts popup it wouldn't hurt to have a new shortcut for when an element is selected to expand the context menu for said element and allow keyboard interaction with the menu. 

Menu's already accessible. This is just to add a new keyboard shortcut that when an element is selected will open that context menu for a user. 

## Pending 
- What should the shortcut be?
Google docs uses:  `⌘ + Shift + \` `Shift + F10` should we do that?

## Acceptance criteria
- open context menu for element select with new keyboard shortcut 
- add shortcut to shortcut popup
- affiliated karma test

**Comment by BrittanyIRL** [comment]
When asking about this issue in our design channel on Slack, @barklund chimed in with some helpful input. Adding here for visibility moving forward in discussion. 

> On Windows, Shift+F10 opens the context menu by default in many programs, so I think it's expected there. But on a Mac there is no such convention, so I don't think users expect it.

> If using a screen-reader, there's often a dedicated shortcut to opening a context-menu that's specific to the screen-reader (though not all non-mouse users are screen-reader users).
In VoiceOver for Mac it's VO+Shift+M (with VO being your dedicated command shortcut for VoiceOver, which by default is Ctrl+Option or Caps Lock - yes, that button).

**Comment by csossi** [comment]
Verified in QA
